dashboard/src/app/(authenticated)/taskDef/[name]/components/TaskDef.tsx

```diff
@@ -1,18 +1,152 @@
+          }}
+        >
+          <option>ALL</option>
+          {Object.keys(TaskStatus)
+            .filter(status => status != TaskStatus.UNRECOGNIZED)
+            .map(status => (
+              <option key={status}>{status}</option>
+            ))}
+        </select>
+      </div>
+      <div className="mb-5 flex max-w-fit items-start justify-between">
+        <Field className="flex items-center justify-between">
+          <Label className="block w-1/2 font-bold">Created after:</Label>
+          <Input
+            type="datetime-local"
+            value={createdAfter}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) => setCreatedAfter(e.target.value)}
+            className="focus:shadow-outline ml-3 w-full appearance-none rounded border px-3 py-2 leading-tight shadow focus:outline-none"
+          />
+        </Field>
+
+        <Field className="ml-10 flex items-center justify-between">
+          <Label className="block w-1/2 font-bold">Created before:</Label>
+          <Input
+            type="datetime-local"
+            value={createdBefore}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) => setCreatedBefore(e.target.value)}
+            className="focus:shadow-outline ml-4 w-full appearance-none rounded border px-3 py-2 leading-tight shadow focus:outline-none"
+          />
+        </Field>
+      </div>
+
+      {isPending ? (
+        <div className="flex min-h-[360px] items-center justify-center text-center">
+          <ArrowPathIcon className="h-8 w-8 animate-spin fill-blue-500 stroke-none" />
+        </div>
+      ) : (
+        <div className="flex min-h-[360px] flex-col gap-4">
+          <table className="text-surface min-w-full text-center text-sm font-light">
+            <thead className="border-b border-neutral-200 bg-neutral-300 font-medium">
+              <tr>
+                <td scope="col" className="px-6 py-4">
+                  WfRun Id
+                </td>
+                <th scope="col" className="px-6 py-4">
+                  Task GUID
+                </th>
+                <th scope="col" className="px-6 py-4">
+                  Creation Date
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+              {data?.pages.map((page, i) => (
+                <Fragment key={i}>
+                  {page.resultsWithDetails.length > 0 ? (
+                    page.resultsWithDetails.map(({ taskRun, nodeRun }) => {
+                      return (
+                        <tr key={taskRun.id?.taskGuid} className="border-b border-neutral-200">
+                          <td className="px-6 py-4">😡
+                            <Link
+                              className="py-2 text-blue-500 hover:underline"
+                              target="_blank"
+                              //? Will have to create a new request to get the nodeName to select the node in the diagram 😡 (will do this in the rewrite)
```

Comment:
left a comment on top of your comment on the PR. Please I think we can remove this comments :)